### ISSUE TITLE
[Bitnami WordPress Multisite] <Issues with Setting Up WildCard Cert>

### ISSUE THREAD

**Issue by rjoel98** [bitnami/vms#607]
### Platform

AWS

### bndiagnostic ID [know more about bndiagnostic ID](https://docs.bitnami.com/general/how-to/understand-bndiagnostic/)

4092a997-4a3f-96cb-88db-863be0d50107

### bndiagnostic output

`[Tue Jan 17 17:09:28.895720 2023] [proxy_fcgi:error] [pid 28708:tid 
139835528402688] [client **ip_address**:56447] AH01071: Got error 'PHP message: 
PHP Deprecated: Return type of Requests_Cookie_Jar::offsetExists($key) should 
either be compatible with ArrayAccess::offsetExists(mixed $offset): bool, or the 
#[\ReturnTypeWillChange] attribute should be used to temporarily suppress the 
notice in /opt/bitnami/wordpress/wp-includes/Requests/Cookie/Jar.php on line 
63PHP message: PHP Deprecated: Return [Tue Jan 17 17:16:23.535562 2023] 
[proxy:error] [pid 28708:tid 139835511617280] (2)No such file or directory: 
AH02454: FCGI: attempt to connect to Unix domain socket 
/opt/bitnami/php/var/run/www.sock (www-fpm:8000) failed
 [Tue Jan 17 17:16:23.537932 2023] [proxy_fcgi:error] [pid 28708:tid 
139835511617280] [client **ip_address**:56520] AH01079: failed to make 
connection to backend: httpd-UDS, referer: 
http://ghomerch.com/wp-admin/my-sites.php
`

### bndiagnostic was not useful. Could you please tell us why?

Did not describe why there was an error presenting the domains to letsencrypt

### Describe your issue as much as you can

I have a new WP multisite that I am launching, currently I am working on getting the SSL cert setup and seem to be having some issues with following this article: [https://aws.amazon.com/premiumsupport/knowledge-center/lightsail-bitnami-wildcard-ssl-cert/](https://aws.amazon.com/premiumsupport/knowledge-center/lightsail-bitnami-wildcard-ssl-cert/)

I have gotten to the point of requesting the certificate, but when I type in the information I get these errors for each domain I am trying to register:  
`acme: cleaning up failed: lightsail: InvalidInputException: These parameters are required: domainName`
`acme: error presenting token: lightsail: InvalidInputException: These parameters are required: domainName`

I have tried looking online on the Bitnami site, Letsencrypt, and AWS with no luck.

Any suggestions would be greatly appreciated!

**Comment by mdhont** [comment]
At which step does the error occur? Can you paste the command and the error?

**Comment by rjoel98** [comment]
Here is the command I used from the article: 
`sudo /opt/bitnami/letsencrypt/lego --email="support@crestoneadvising.com" --domains="ghomerch.com" --domains="*.ghomerch.com" --dns lightsail --path="/opt/bitnami/letsencrypt" run'

And this is the full error output that I received after accepting the TOS and the system registered my email:
'2023/01/17 17:19:30 [INFO] [*.ghomerch.com] acme: use dns-01 solver
2023/01/17 17:19:30 [INFO] [ghomerch.com] acme: Could not find solver for: tls-alpn-01
2023/01/17 17:19:30 [INFO] [ghomerch.com] acme: Could not find solver for: http-01
2023/01/17 17:19:30 [INFO] [ghomerch.com] acme: use dns-01 solver
2023/01/17 17:19:30 [INFO] [*.ghomerch.com] acme: Preparing to solve DNS-01
2023/01/17 17:19:31 [INFO] [ghomerch.com] acme: Preparing to solve DNS-01
2023/01/17 17:19:31 [INFO] [*.ghomerch.com] acme: Cleaning DNS-01 challenge
2023/01/17 17:19:31 [WARN] [*.ghomerch.com] acme: cleaning up failed: lightsail: InvalidInputException: These parameters are required: domainName
{
  RespMetadata: {
    StatusCode: 400,
    RequestID: "945faa44-dce4-461e-922f-70d20b8b9851"
  },
  Code_: "MissingParams",
  Message_: "These parameters are required: domainName"
} 
2023/01/17 17:19:31 [INFO] [ghomerch.com] acme: Cleaning DNS-01 challenge
2023/01/17 17:19:32 [WARN] [ghomerch.com] acme: cleaning up failed: lightsail: InvalidInputException: These parameters are required: domainName
{
  RespMetadata: {
    StatusCode: 400,
    RequestID: "876a86ca-69ea-422f-965b-509ef4e81d7d"
  },
  Code_: "MissingParams",
  Message_: "These parameters are required: domainName"
} 
2023/01/17 17:19:32 [INFO] Deactivating auth: https://acme-v02.api.letsencrypt.org/acme/authz-v3/195964782317
2023/01/17 17:19:32 [INFO] Deactivating auth: https://acme-v02.api.letsencrypt.org/acme/authz-v3/195964782327
2023/01/17 17:19:32 Could not obtain certificates:
	error: one or more domains had a problem:
[*.ghomerch.com] [*.ghomerch.com] acme: error presenting token: lightsail: InvalidInputException: These parameters are required: domainName
{
  RespMetadata: {
    StatusCode: 400,
    RequestID: "151a3239-c5d5-4fe6-b617-8de732e5823b"
  },
  Code_: "MissingParams",
  Message_: "These parameters are required: domainName"
}
[ghomerch.com] [ghomerch.com] acme: error presenting token: lightsail: InvalidInputException: These parameters are required: domainName
{
  RespMetadata: {
    StatusCode: 400,
    RequestID: "e54d5e88-fe97-4bae-9e4b-921192967518"
  },
  Code_: "MissingParams",
  Message_: "These parameters are required: domainName"
}
`

**Comment by mdhont** [comment]
I recommend checking the lego documentation regarding lightsail dns: https://go-acme.github.io/lego/dns/lightsail/

**Comment by rjoel98** [comment]
Thank you for that suggestion, I did look over that page already actually to ensure I was doing things as close to the recommended rought as possible. Unfortunately I did not find anything helpful, I might try to submit a ticket with them though to see what they say but I figured they would most likely refer me to you all since they say anything to do with bitnami not supported by then directly. I'll keep you updated though, thank you!
